requirements.txt

```diff
@@ -1,4 +1,4 @@
-tifffile>=2021.7.2,<2022.4.28
+tifffile>=2023.4.12
```

Comment:
Does opening the pin for `>=2021.7.2` do the job, or are there regressions that were introduced in version `2022.4.28`?

Reply:
Never mind, I see the issue now. Perhaps we can mark geotiff to skip `tiffile==2022.4.28`?

Reply:
```suggestion
tifffile>=2021.7.4,!=2022.4.28
```